src/Pwa/PageLoader/Context/PathResolver.php

```diff
@@ -29,68 +23,50 @@ class PathResolver implements PathResolverInterface
     private const ROOT_ROUTE_NAME = PageController::NAVIGATION_PAGE_ROUTE;
 
     /**
-     * @var SalesChannelRouteRepository
+     * @var SeoResolverInterface
      */
-    private $routeRepository;
+    private $seoResolver;
 
-    /**
-     * @var EntityRepositoryInterface
-     */
-    private $salesChannelRepository;
-
-
-    public function __construct(SalesChannelRouteRepository $routeRepository, EntityRepositoryInterface $salesChannelRepository)
+    public function __construct(SeoResolverInterface $seoResolver)
```

Comment:
DI is breaking here, as it doesn't get injected properly. (We don't use autowiring for service arguments).